examples/experimental/openvino/yolo_v5/requirements.txt

```diff
@@ -0,0 +1 @@
+yolov5
```

Comment:
Should we download the `yolov5` repo in this case?

Reply:
Could you suggest changes?

Reply:
I mean if we installed `yolov5` through the pip is it required to clone the yolov5 repo or not?